source/utilities.tex

```diff
@@ -670,7 +671,7 @@
 \tcode{static_cast<underlying_type_t<T>>(value)}.
 \end{itemdescr}
 
-\rSec2[utility.unreachable]{Function \tcode{unreachable}}
+\rSec2[utility.undefined]{Undefined behavior}
```

Comment:
This needs a \movedxref entry in xrefdelta.tex

Reply:
xrefprev seems to be missing utility.unreachable.   I'll add it, but it also seems to be missing about 500 other stable names from N4950 that are not in xrefprev.  would you like me to add those?  (It's unclear to me what the intent of the contents of xrefprev is --- whether it should be removed names, or just all names that were previously in use -- either way that is not what it is).

Reply:
ok.  shouldn't it be the list of stable names from any previously published standard?  Or from C++17 to C++23 inclusive?  I could grab that pretty easily if it helps (I already extracted it from C++23), but for this PR I have added utility.uncreachable manually to xrefprev.

Reply:
The problem is we've added the label since C++17, and now we're renaming it.  So, it's technically not correct that it was a label in C++17, which xrefprev attempts to represent.

The once-off fix feels reasonable for now.

Reply:
My understanding of @StephanTLavavej's request is that we don't just want the diff of the stable labels relative to one arbitrary point in the past, but rather, we want a full history of things that have happened, so that if you've been working with any one version of the document and are switching to a newer one, you can look up what happened to the label you had been working with.

Reply:
I definitely think that'd be way more useful than just "here's things that were in C++17 that might have moved".    It might be even more useful to have the index list when the name was moved (i.e, "utility.unreachable (moved in C++26, see utility.undefined)"